### DISCUSSION TITLE
QNH not flashing above TRANS ALT

### DISCUSSION THREAD

**Discussion by Martievo** [flybywiresim/aircraft#3253] [Help]
Hello, 

In the PFD the QNH is not flashing anymore when above the set TRANS Alltitude. It is not working in the Stable and DEV.
Tried with no other mods same Result. 

Please help.

**Comment by tracernz** [comment]
Hmmm, just finished a flight and it blinked at me during both climb and descent.

**Reply by Martievo** [reply]
It Worked fine a few days back.
